### DISCUSSION TITLE
zh/guide/drivers/UrlTree

### DISCUSSION THREAD

**Discussion by giscus** [AlistGo/docs#206] [Announcements]
# zh/guide/drivers/UrlTree

地址树-UrlTree 是什么? 是用来挂载单个文件链接的，详情可以查看 →→→原需求说明 (https://github.com/alist-org/alist/issues/3268)←←← 用法 具体的用法大概讲解一下，建议将下面不同用法都看看越往后越进阶玩法 别看写的看起来很难很啰嗦实际上自己操作一下秒懂。 tab 1 最简单的用法，直接写链接...

https://alist.nn.ci/zh/guide/drivers/UrlTree.html

**Comment by tophyZH** [comment]
这个功能不错，我正考虑在AI的帮助下搞一个网站地图的导航页，就是类似一个网站的内容报告一样的首页，实现对网站的鸟瞰，我想这个功能就很接近了。
如果可以有一个脚本或者另一个同级的、功能性的存储类型，或者一个可以调用的变量或参数，能够实现对目录结构的遍历（如bleve）和这个功能的“Url结构块”结合/关联，是不就更完美了。
#不懂编程的程序员#结果导向

**Reply by Soltus** [reply]
要是支持本地路径就好了
